docs/components/board/pi.md

```diff
@@ -36,9 +36,25 @@ Edit and fill in the attributes as applicable.
 {
   "components": [
     {
-      "type": "board",
+      "name": "<your-pi-board-name>",
       "model": "pi",
-      "name": "<board_name>"
+      "type": "board",
+      "namespace": "rdk",
+      "attributes": {
+        "analogs": [
+          <...See table below...>
```

Comment:
```suggestion
          <See table below>
```
(opt) I think the ... is a little informal but not a biggie, would suggest removing

Reply:
I put it in because you (or maybe someone else) used it in various other places like the base configs. I figured this is a way to emphasize that this needs to be expanded significantly and isn't just a one-line edit for the user to make.

Reply:
I'm not anti using ... at all I just don't think it is needed in this particular context on both sides of the brackets, if you prefer happy to keep it in though!